crud.go

```diff
@@ -393,6 +401,9 @@ func (bucket *WalrusBucket) waitAfterWrite(seq uint64, opt sgbucket.WriteOptions
 func (bucket *WalrusBucket) write(k string, exp uint32, raw []byte, opt sgbucket.WriteOptions) (seq uint64, err error) {
 	bucket.lock.Lock()
 	defer bucket.lock.Unlock()
+	if len(raw) > 20000000{
+		return 0, DocTooLargeError{}
```

Comment:
We don't need to define a new error type unless it's being explicitly checked elsewhere, or have multiple places it can be returned from.

`errors.New("document too large")` should suffice